src/compiler/scala/tools/nsc/transform/CleanUp.scala

```diff
@@ -524,4 +597,24 @@ abstract class CleanUp extends Statics with Transform with ast.TreeDSL {
 
   } // CleanUpTransformer
 
+
+  val BOX = TermName("box")
+  val objectMethods = Map[Name, TermName](
+    TermName("hashCode") -> TermName("hashCode"),
+    TermName("toString") -> TermName("toString")
+    )
+  val doubleAndFloatRedirectMethods = Map[Name, TermName](
+    TermName("isNaN") -> TermName("isNaN"),
+    TermName("isInfinite") -> TermName("isInfinite")
+    )
+  val javaNumberConversions         = Map[Name, TermName](
+    TermName("byteValue") -> TermName("toByte"),
+    TermName("shortValue") -> TermName("toShort"),
+    TermName("intValue") -> TermName("toInt"),
+    TermName("longValue") -> TermName("toLong"),
+    TermName("floatValue") -> TermName("toFloat"),
+    TermName("doubleValue") -> TermName("toDouble")
```

Comment:
The RHS ones exist as names, as do hashCode and toString (as `hashCode_` and `toString_`).  Should we add the others too?  I'm not sure what all the reasons or principles for using or not using StdNames, but at a guess I think it's better to have them there rather than here.

Reply:
there was no particular reason for putting then here. Happy to move
presume I should move the `TermName`s, not the Map

Reply:
Yep - the mapping is specific to the rewrite here.